openmc/mgxs/mgxs.py

```diff
@@ -1366,12 +1366,9 @@ def get_pandas_dataframe(self, groups='all', nuclides='all',
         xs_type: {'macro', 'micro'}
             Return macro or micro cross section in units of cm^-1 or barns.
             Defaults to 'macro'.
-        summary : None or openmc.Summary
-            An optional Summary object to be used to construct columns for
-            distribcell tally filters (default is None). The geometric
-            information in the Summary object is embedded into a multi-index
-            column with a geometric "path" to each distribcell intance.
-            NOTE: This option requires the OpenCG Python package.
+    distribcell_paths : list of str
+        The paths traversed through the CSG tree to reach each distribcell
+        instance (for 'distribcell' filters only)
```

Comment:
Indentation is wrong on this block. Also, please indicate "optional" for arguments that have default values and list their defaults in the description.